Classes/Flownative/Aws/S3/S3Target.php

```diff
@@ -337,9 +337,9 @@ protected function publishFile($sourceStream, $relativeTargetPathAndFilename, Re
 	 */
 	protected function getRelativePublicationPathAndFilename(ResourceMetaDataInterface $object) {
 		if ($object->getRelativePublicationPath() !== '') {
-			$pathAndFilename = $object->getRelativePublicationPath() . $object->getFilename();
+			$pathAndFilename = $object->getRelativePublicationPath() . rawurlencode($object->getFilename());
```

Comment:
Just wondering: Shouldn't the ``$object->getRelativePublicationPath()`` be encoded, too?